app/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,7 @@
     <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.USE_BIOMETRIC"/>
```

Comment:
Why this permission? 🤔

Reply:
pardon my naivete , this wasn't supposed to be pushed in this PR :( I will fix it